lxd/device/disk.go

```diff
@@ -667,7 +667,7 @@ func (d *disk) createDevice() (string, error) {
 	}
 
 	// Check if the source exists.
-	if !shared.PathExists(srcPath) {
+	if !(strings.HasPrefix(fsName, "ceph:") || strings.HasPrefix(fsName, "cephfs:")) && !shared.PathExists(srcPath) {
```

Comment:
srcPath for `ceph:` is actually correct, it's only for the `cephfs:` case where it's not.

I think it'd be best to change the check to be `if fsName != 'ceph' && !shared.PathExists(srcPath)` so we only handle the cephfs case there.

Reply:
My mistake, I had copied over some of our old checks and forgotten the distinction. Fixing now